lib/web_ui/lib/src/engine/semantics/semantics.dart

```diff
@@ -968,6 +969,30 @@ class SemanticsObject {
     }
   }
 
+  void _resetElementOffsets() {
+    if (isMacOrIOS) {
+      if (isDesktop) {
+        element.style
+          ..transformOrigin = '0 0 0'
+          ..transform = 'translate(0px, 0px)';
+      } else {
+        element.style
+          ..top = '0px'
+          ..left = '0px';
```

Comment:
What happens if we use top/left in all browsers?

Reply:
The idea was that layout faster with transform.